manifest-build-tools/application/update_changelog.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python
+# Copyright 2015-2016, EMC, Inc.
+
+"""
+The script update changelog under debian
+
+usage:
+./on-tools/manifest-build-tools/HWIMO-BUILD on-tools/manifest-build-tools/application/update_changelog.py \
+--build-dir d/ \
+--version 1.2.6 \
+--publish \
+--git-credential https://github.com,GITHUB \
+--message "new branch 1.2.6"
+
+The required parameters: 
+build-dir: The top directory which stores all the cloned repositories
+version: The new release version
+git-credential: url, credentials pair for the access to github repos.
+
+The optional parameters:
+message (default value is "new release" + version )
+publish: If true, the updated changlog will be push to github.
+git-credential: url, credentials pair for the access to github repos.
+                If publish is true, the parameter is required.
+"""
+import os
+import sys
+import argparse
+import datetime
+import subprocess
+from RepositoryOperator import RepoOperator
+from common import *
+
+class ChangelogUpdater(object):
+    def __init__(self, repo_dir, version):
+        """
+        The module updates debian/changelog under the directory of repository
+        __repo_dir: the directory of the repository
```

Comment:
There are two underscores here, but the real argument has only one underscore. Please keep them matched.